### PR TITLE
Add SDBits provider

### PR DESCRIPTION
**Pull request by duramato**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: medusa/providers/torrent/html/sdbits.py [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+        # URLs
+        self.url = 'http://sdbits.org'
+        self.urls = {
+            'login': urljoin(self.url, 'takeloginn3.php'),
+            'search': urljoin(self.url, 'browse.php'),
+        }
+
+        # Proper Strings
+        self.proper_strings = ['']
```
Comment (medariox) [inline medusa/providers/torrent/html/sdbits.py]: Why the empty string here?

File: medusa/providers/torrent/html/sdbits.py [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+                    torrent_size = cells[5].get_text(' ')
+                    size = convert_size(torrent_size) or -1
+
+                    pubdate_raw = cells[4].get_text('_').split('_')
+                    if len(pubdate_raw) == 2:
+                        pubdate_raw = parse(pubdate_raw[0]) + parse(pubdate_raw[1])
+                    else:
+                        pubdate_raw = parse(pubdate_raw[0])
+                    pubdate = str(datetime.datetime.now() - datetime.timedelta(seconds=pubdate_raw)) if pubdate_raw else None
```
Comment (medariox) [inline medusa/providers/torrent/html/sdbits.py]: Why the str() here?

File: medusa/providers/torrent/html/sdbits.py (hunk 1 above)
Comment (duramato) [inline medusa/providers/torrent/html/sdbits.py]: There are no propers and such, and if there are a normal search returns them. No need for 3 more searches.

File: medusa/providers/torrent/html/sdbits.py (hunk 2 above)
Comment (duramato) [inline medusa/providers/torrent/html/sdbits.py]: String representation of the date object.

**Pull request merged**